frontend/src/theme/mediaQueries.js

```diff
@@ -0,0 +1,108 @@
+/* EXAMPLE OF USE IN COMPONENT
+import {Device, Values} from "../../theme/mediaQueries";
+@media ${Device.tablet} {
+  ${Values.fontSize14}
+  ${Values.width100}
+}
+*/
+
+/* CHANGED FILES WITH MEDIAQUERIES */
+// DONE /components /composed /AdCardList /AdCardList.styles.js
+// DONE /components /composed /Card /Card.style.js
+// DONE /components /composed /Charts /BarChart /BarGraphic.styles.js
+// DONE /components /composed /Charts /LineChart /LineChart.styles.js
+// DONE /components /composed /Gallery /Gallery.styles.js
+// DONE /components /composed /GlobalFilters /GlobalFilters.styles.js
+// DONE /components /composed /SearchBar /SearchBarStyles.js
+// DONE /components /layout /Footer /Footer.styles.js
+// DONE /components /units /inputNumber /InputNumber.styles.js
+// DONE /components /units /Notifications /Notifications.styles.js
+// DONE /components /units /SearchButton /SearchButtonStyles.js
+// DONE /screens /Ad /Ad.styles.js
+// DONE /screens /AdList /AdCard /AdCard.style.js
+// DONE /screens /AdList /AdListFilter /AdListFilter.style.js
+// DONE /screens /AdList /AdList.style.js
+// DONE /screens /CreateNewAd /CreateNewAd.styles.js
+// DONE /screens /Dashboard /Dashboard.style.js
+// DONE /screens /EditAd /EditAd.styles.js
+// DONE /screens /ListaUsuariosAdmins /ListaUsuariosAdmins.styles.js
+// DONE /screens /MyBills /Bill.styles.js
+// DONE /screens /MyBills /MyBills.styles.js
+// DONE /screens /Search /Search.style.js
+// DONE /screens /UserFlow /Profile /Profile.styles.js
+
+// Screen Sizes
+const responsiveSizes = {
```

Comment:
Simplify to have less sizes: https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/

Reply:
Understood. But what happens with customs? I should override the custom uses along the project and study case by case? I think in some situations custom number mediaqueries were used to fix bad templating behaviors.